dronekit/__init__.py

```diff
@@ -2748,6 +2794,65 @@ def __setitem__(self, index, value):
         self._vehicle._wpts_dirty = True
 
 
+class ServoOutputRaw(object):
+    """
+    The RAW servo readings from the OUTPUT of the autopilot
+    This contains the true raw values without any scaling to allow data capture and system debugging.
+
+    The message definition is here: https://pixhawk.ethz.ch/mavlink/#SERVO_OUTPUT_RAW
+
+
+    """
+
+    def __init__(self,
+                 time_boot_us=None, # Timestamp (microseconds since system boot)
+                 port=None,         # Servo output port (set of 8 outputs = 1 port). Most MAVs will just use one, but this allows to encode more than 8 servos.
+                 servo1_raw=None,   # Servo output 1 value, in microseconds
+                 servo2_raw=None,   # Etc..
+                 servo3_raw=None,
+                 servo4_raw=None,
+                 servo5_raw=None,
+                 servo6_raw=None,
+                 servo7_raw=None,
+                 servo8_raw=None,
+                 servo9_raw=None,
+                 servo10_raw=None,
+                 servo11_raw=None,
+                 servo12_raw=None,
+                 servo13_raw=None,
+                 servo14_raw=None,
+                 servo15_raw=None,
+                 servo16_raw=None):
+
+        self.time_boot_us = time_boot_us
+        self.port = port
+        self.servo1_raw = servo1_raw
+        self.servo2_raw = servo2_raw
+        self.servo3_raw = servo3_raw
+        self.servo4_raw = servo4_raw
+        self.servo5_raw = servo5_raw
+        self.servo6_raw = servo6_raw
+        self.servo7_raw = servo7_raw
+        self.servo8_raw = servo8_raw
+        self.servo9_raw = servo9_raw
+        self.servo10_raw = servo10_raw
+        self.servo11_raw = servo11_raw
+        self.servo12_raw = servo12_raw
+        self.servo13_raw = servo13_raw
+        self.servo14_raw = servo14_raw
+        self.servo15_raw = servo15_raw
+        self.servo16_raw = servo16_raw
+
+    def __str__(self):
+        """
+        String representation used to print the ServoOutputRaw object.
+        """
+        return "SERVO_OUTPUT_RAW: time_boot_us={},port={},servo1_raw={},servo2_raw={},servo3_raw={},servo4_raw={},servo5_raw={},servo6_raw={},servo7_raw={},servo8_raw={},servo9_raw={},servo10_raw={},servo11_raw={},servo12_raw={},servo13_raw={},servo14_raw={},servo15_raw={},servo16_raw={}".format(
+            self.time_boot_us, self.port, self.servo1_raw, self.servo2_raw, self.servo3_raw, self.servo4_raw,
+            self.servo5_raw, self.servo6_raw, self.servo7_raw, self.servo8_raw, self.servo9_raw, self.servo10_raw,
+            self.servo11_raw, self.servo12_raw, self.servo13_raw, self.servo14_raw, self.servo15_raw, self.servo16_raw)
```

Comment:
We have been using the `__str__` decorator for debugging purposes on all of our top-level classes, do you think this is the most useful representation of data we can get out of this class?, would you typically expect all of the variables to be set and by reading make something out of it?, what if we are logging this, do you think this is the appropriate way to read it?